src/Pages/Professionals/AllProfessionals/ProfessionalsTable.js

```diff
@@ -0,0 +1,112 @@
+import React from "react"
+import { UserType } from "types"
+import Table from "components/Table"
+import { NavLink } from "react-router-dom"
+import { css } from "emotion"
+import PropTypes from "prop-types"
+import { useI18n } from "providers/I18n"
+
+const ProfessionalsTable = ({ resolvedData, pageInfo, setPageInfo }) => {
+  return (
+    <div>
+      <Table
+        onChange={setPageInfo}
+        columns={columns}
+        data={resolvedData.data}
+        total={resolvedData.total}
+        pageIndex={pageInfo.pageIndex}
+        pageSize={pageInfo.pageSize}
+      />
+    </div>
+  )
+}
+
+export default ProfessionalsTable
+
+ProfessionalsTable.propTypes = {
+  users: UserType,
+  resolvedData: UserType,
+  setPageInfo: PropTypes.func,
+  pageInfo: PropTypes.object
+}
+
+NameColumn.propTypes = {
+  value: UserType
+}
+
+function NameColumn({ value }) {
+  const { firstName, lastName } = value
+  return <div>{`${firstName} ${lastName}`}</div>
+}
+
+LastVisitColumn.propTypes = {
+  value: UserType
+}
+
+function LastVisitColumn({ value }) {
+  const { patient, date } = value.lastvisit
+
+  return (
+    <div>
+      {patient}
+      <div className={subrow}>{date}</div>
+    </div>
+  )
+}
+
+ActionsColumn.propTypes = {
+  value: UserType
+}
+
+function ActionsColumn({ value }) {
+  const { id } = value
+  return (
+    <div>
+      <NavLink to={`professionals/details/${id}`} className={viewAction}>
+        View
+      </NavLink>
+    </div>
+  )
+}
+
+const Translate = ({ arg }) => {
+  const { t } = useI18n()
+  return t(`${arg}`)
+}
+
+export const columns = [
+  {
+    Header: <Translate arg={"int.name"} />,
+    accessor: (row) => row,
+    id: 1,
```

Comment:
What is this id field here for?

Reply:
It's needed from react table as identification because accessor (which is the main identification) is a function.